src/messages/rdf.js

```diff
@@ -0,0 +1,26 @@
+import { gettext as _, singleLineString } from 'utils';
+
+
+export var _tagNotAllowed = (tagName) => {
+  return {
+    code: `TAG_NOT_ALLOWED_${tagName.toUpperCase()}`,
+    message: _(`The <${tagName}> tag is not allowed`),
+    description: _(singleLineString`Your RDF file contains the <${tagName}> tag,
+      which is not allowed in an Add-on.`),
+  };
+};
+
+export var _tagObsolete = (tagName) => {
+  return {
+    code: `TAG_OBSOLETE_${tagName.toUpperCase()}`,
+    message: _(`The <${tagName}> tag is obsolete`),
+    description: _(singleLineString`Your RDF file contains the <${tagName}> tag,
+      which is obsolete.`),
+  };
+};
+
+export const TAG_NOT_ALLOWED_HIDDEN = _tagNotAllowed('hidden');
```

Comment:
I'd like that, but I didn't think one could do `export const $VARIABLE_VARIABLE_NAME = whatever()`.

It was my instinct coming from Ruby :smile: . But I feel like ES6 can't do it? We could make them keys off an object, but then messages become less consistent in their usage (from `TAG_NOT_ALLOWED_HIDDEN` to `TAG_NOT_ALLOWED.HIDDEN`).